src/App.js

```diff
@@ -1,27 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] Please as it is a requirement for this project, all your components should be class based, please convert you `App.js` component to a class-based component. See screenshot of the requirements below for clarifications.

![ClassBased](https://user-images.githubusercontent.com/65852621/154005009-23490573-f4ae-49d2-bd93-d551f43faf15.png)

Reply:
Resolved